.circleci/config.yml

```diff
@@ -0,0 +1,197 @@
+              libtokyocabinet-dev \
+              libdb-dev \
+              libqdbm-dev \
+              libjpeg-dev \
+              libpng-dev \
+              libfreetype6-dev
+      - run:
+          name: ./configure
+          command: |
+            ./buildconf -f
+            ./configure \
+              --enable-debug \
+              --enable-zts \
+              --enable-option-checking=fatal \
+              --prefix=/usr \
+              --enable-phpdbg \
+              --enable-fpm \
+              --enable-opcache \
+              --with-pdo-mysql=mysqlnd \
+              --with-mysqli=mysqlnd \
+              --with-pgsql \
+              --with-pdo-pgsql \
+              --with-pdo-sqlite \
+              --enable-intl \
+              --without-pear \
+              --enable-gd \
+              --with-jpeg \
+              --with-webp \
+              --with-freetype \
+              --with-xpm \
+              --enable-exif \
+              --with-zip \
+              --with-zlib \
+              --with-zlib-dir=/usr \
+              --enable-soap \
+              --enable-xmlreader \
+              --with-xsl \
+              --with-tidy \
+              --enable-sysvsem \
+              --enable-sysvshm \
+              --enable-shmop \
+              --enable-pcntl \
+              --with-readline \
+              --enable-mbstring \
+              --with-curl \
+              --with-gettext \
+              --enable-sockets \
+              --with-bz2 \
+              --with-openssl \
+              --with-gmp \
+              --enable-bcmath \
+              --enable-calendar \
+              --enable-ftp \
+              --with-pspell=/usr \
+              --with-enchant=/usr \
+              --with-kerberos \
+              --enable-sysvmsg \
+              --with-ffi \
+              --enable-zend-test \
+              --enable-dl-test=shared \
+              --with-ldap \
+              --with-ldap-sasl \
+              --with-password-argon2 \
+              --with-mhash \
+              --with-sodium \
+              --enable-dba \
+              --with-cdb \
+              --enable-flatfile \
+              --enable-inifile \
+              --with-tcadb \
+              --with-lmdb \
+              --with-qdbm \
+              --with-snmp \
+              `#--with-unixODBC` \
+              --with-imap \
+              --with-kerberos \
+              --with-imap-ssl \
+              `#--with-pdo-odbc=unixODBC,/usr` \
+              `#--with-pdo-oci=shared,instantclient,/opt/oracle/instantclient` \
+              `#--with-oci8=shared,instantclient,/opt/oracle/instantclient` \
+              --with-config-file-path=/etc \
+              --with-config-file-scan-dir=/etc/php.d \
+              --with-pdo-firebird \
+              `#--with-pdo-dblib` \
+              --disable-phpdbg \
+              `#--enable-werror`
+      - run:
+          name: make
+          command: make -j$(/usr/bin/nproc) > /dev/null
```

Comment:
We had issue with that in my other job. Think nproc still does not work correctly in circleci: see https://circleci.canny.io/cloud-feature-requests/p/have-nproc-accurately-reporter-number-of-cpus-available-to-container .

Reply:
That's good to know. Cirrus had the same issue. I changed it to a hard-coded `2`.